src/components/styled-radio-group/radio-group/RadioGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useMemo } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { RadioItem } from './RadioItem';
+import { RadioGroupProps } from './types';
+
+export function RadioGroup({ containerStyle, layout = 'column', onPress, items, selectedId }: RadioGroupProps) {
+  const handlePress = (id: string) => {
+    if (id !== selectedId && onPress) {
+      onPress(id);
+    }
+  };
+
+  const itemsLocal = useMemo(
+    () =>
+      items.map(item => ({
+        key: item.id,
+        ...item,
+        selected: item.id === selectedId,
+        onPress: (id: string) => {
+          handlePress(id);
+
+          if (item.onPress) {
+            item.onPress(id);
+          }
+        }
+      })),
+    [items, selectedId]
+  );
+
+  return (
+    <View style={[styles.container, { flexDirection: layout }, containerStyle]}>
```

Comment:
I think we will have 1 direction for now. You could just add default value into styles